_data/abbr.yml

```
@@ -1,3 +1,4 @@
 EZSP: EmberZnet serial protocol
+MFA: multi-factor authentication
```

```suggestion
MFA: Multi-factor authentication
```
This will be in its own separate box when you hover over it, so it should be capitalised as it is the start.